r/R/parquet.R

```diff
@@ -432,19 +453,45 @@ ParquetFileReader <- R6Class("ParquetFileReader",
   active = list(
     num_rows = function() {
       as.integer(parquet___arrow___FileReader__num_rows(self))
+    },
+    num_columns = function() {
+      parquet___arrow___FileReader__num_columns(self)
+    },
+    num_row_groups = function() {
+      parquet___arrow___FileReader__num_row_groups(self)
     }
   ),
   public = list(
-    ReadTable = function(col_select = NULL) {
-      col_select <- enquo(col_select)
-      if (quo_is_null(col_select)) {
+    ReadTable = function(column_indices = NULL) {
+      if (is.null(column_indices)) {
         shared_ptr(Table, parquet___arrow___FileReader__ReadTable1(self))
+      } else if (is_integerish(column_indices)) {
```

Comment:
Hmm, the others don't check is_integerish, and if you do check that here, you have a missing `else` case to add. Presumably `vec_cast` will error for you if you give it something that isn't reasonably an integer?

```suggestion
      } else {
```